src/smif/decision/decision.py

```diff
@@ -62,20 +65,32 @@ def _set_up_decision_modules(self, modelrun_name):
 
         # Read in strategies
         strategies = self._store.read_strategies(modelrun_name)
-
         self.logger.info("%s strategies found", len(strategies))
         planned_interventions = []
         planned_interventions.extend(initial_conditions)
 
         for index, strategy in enumerate(strategies):
             # Extract pre-specified planning interventions
-            if strategy['strategy'] == 'pre-specified-planning':
+            if strategy['name'] == 'pre-specified-planning':
 
                 msg = "Adding %s planned interventions to pre-specified-planning %s"
                 self.logger.info(msg, len(strategy['interventions']), index)
 
                 planned_interventions.extend(strategy['interventions'])
 
+            else:
```

Comment:
Maybe be explicit here that this logic deals with strategy['name'] == 'agent'

Reply:
The exception is only for pre-specified planning modules, all other module types (rule-based or optimisation) should be loaded from the python script.